spellcheck/src/Makefile.rel.am

```diff
@@ -0,0 +1,29 @@
+if ENABLE_SPELLCHECK
+geanyplugins_LTLIBRARIES += %D%/spellcheck.la
+
+%C%_spellcheck_la_SOURCES = \
+	%D%/scplugin.h \
+	%D%/speller.h \
+	%D%/gui.h \
+	%D%/gui.c \
+	%D%/speller.c \
+	%D%/scplugin.c
+
+%C%_spellcheck_la_CFLAGS = \
+	$(AM_CFLAGS) \
+	$(ENCHANT_CFLAGS)
+
+if HAVE_ENCHANT_1_5
+%C%_spellcheck_la_CFLAGS += -DHAVE_ENCHANT_1_5
+endif
+
+%C%_spellcheck_la_LIBADD = \
+	$(COMMONLIBS) \
+	$(ENCHANT_LIBS)
+
+%C%_spellcheck_la_CPPCHECKFLAGS = \
+	$(AM_CPPCHECKFLAGS) \
+	-DSCE_PAS_DEFAULT=0
```

Comment:
That's cute, but `AM_CPPCHECKFLAGS` is not a real Automake variable, and we don't support per-target versions of it.  I have no good solution to propose here, but maybe having the cppcheck snippet define plugin-specific targets that get added to check-local, something like

```make
check-cppcheck-$(plugin): $(something clever) ...
check-local: check-cppcheck-$(plugin)
```

That'd probably work just fine for both recursive and non-recursive versions, and "only" requires additionally having `$(plugin)` defined -- but for the `$(something clever)` part I'm not certain how to handle… maybe defining `reldir=%D%` in the caller and using that somehow?